apps/src/templates/codeDocs/ProgrammingExpressionOverview.jsx

```diff
@@ -22,21 +28,49 @@ export default function ProgrammingExpressionOverview({programmingExpression}) {
         VIDEO_HEIGHT
       );
     }
-  });
+    if (titleRef.current && programmingExpression.blockName) {
```

Comment:
`programmingExpression` is a prop, which can change between renders (we just don't expect it to for our case here), so it should be a dependency of this `useEffect`. we can keep this new code in the existing `useEffect` because `programmingExpression` should've been a dependency of the existing code as well (since it depends on the value of `programmingExpression.video`)

Reply:
ah good catch thanks!